packages/plugins/videos-ui/src/lib/pages/video-detail-page/video-detail-page.component.html

```
@@ -1,10 +1,44 @@
+				@defer(on viewport) {
+				<div nbInfiniteList [threshold]="500" [throttleTime]="300" (bottomThreshold)="fetchMoreVideos()"
+					class="list">
+					<plug-video-list [vertical]="true"></plug-video-list>
+				</div>
+				}@placeholder{
+				<ngx-no-data-message class="no-data" status="danger"
+					message="Loading camshots..."></ngx-no-data-message>
+				}
+			</nb-tab>
+			}
+			<nb-tab tabTitle="Camshots">
```

HTMLHint (1.5.0)

[error] The attribute name of [ tabTitle ] must be in lowercase.

(attr-lowercase)